packages/cli/src/run.test.ts

```diff
@@ -34,6 +34,7 @@ describe("Run command", () => {
       expect(code).toBe(exitCode);
       expect(stripAnsi(stdout.toString())).toMatchSnapshot("stdout");
       expect(stripAnsi(stderr.toString())).toMatchSnapshot("stderr");
-    }
+    },
+    10_000 /* 10 seconds */
```

Comment:
that seems excessive 😅 perhaps it was just committed after debugging something? is there any reason why those tests would be taking so much time?

```suggestion
    }
```

Reply:
On my mac, just running `node packages/cli/bin.js` takes almost 5 seconds just to print out an error, so the default timeout of 5 seconds is way too short to reliably run the unit tests. Not sure if this is misconfiguration and/or node versions or whatever, but starting up preconstruct takes a very long time.

Reply:
Hmm, perhaps `it.each` is counted as a single test and the timeout is shared across all cases. Maybe using `it.concurrent.each` would fix this? 🤔 I can dig into this later to confirm how the timeout is calculated

Reply:
I don't think the issue is with the tests -- what I'm saying is, let's say I just sit at a regular Terminal, booting up preconstruct takes an extra 5 real seconds compared to running a pre-compiled (ts -> js) file. It takes about 5 seconds to show the first line of console output.

So, there's nothing "wrong" with Jest -- running the tool takes ~5 seconds, so every individual test has a 50% chance of failing because it's right on the edge of the timeout.

Reply:
hm, ok - let's roll with that. We probably could speed this up significantly by not going through `spawn` and co. We could invoke "command functions" directly in tests